include/direwolf/vulkan/vulkancommons.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+#include "vulkan/vulkan.h"
+
+#include <iostream>
+#include <string>
+
+#if defined (_WIN32)
+#include <windows.h>
+#endif // _WIN32
+
+namespace dw::vulkan {
+
+#if defined (_WIN32)
+typedef HMODULE VulkanRTLPtr;
+#elif defined (__linux) || defined (__APPLE__)
+typedef void* VulkanRTLPtr;
+#else
+typedef void* VulkanRTLPtr;
```

Comment:
what this case? should never happen?

Reply:
I wanted it in there just for modularity, but sure, I'll remove it